common/crypto/openssl/ec.c

```diff
@@ -440,6 +595,93 @@ oe_result_t oe_ec_public_key_from_coordinates(
 
     return result;
 }
+#else
+oe_result_t oe_ec_public_key_from_coordinates(
+    oe_ec_public_key_t* public_key,
+    oe_ec_type_t ec_type,
+    const uint8_t* x_data,
+    size_t x_size,
+    const uint8_t* y_data,
+    size_t y_size)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    oe_public_key_t* impl = (oe_public_key_t*)public_key;
+    int nid;
+    EVP_PKEY* pkey = NULL;
+    EVP_PKEY_CTX* ctx = NULL;
+    OSSL_PARAM_BLD* param_bld = NULL;
+    OSSL_PARAM* params = NULL;
+
+    if (public_key)
+        oe_secure_zero_fill(public_key, sizeof(oe_ec_public_key_t));
+
+#if !defined(OE_BUILD_ENCLAVE)
+    /* Initialize OpenSSL */
+    oe_crypto_initialize();
+#endif
+
+    /* Reject invalid parameter */
+    if (!public_key || !x_data || !x_size || x_size > OE_INT_MAX || !y_data ||
+        !y_size || y_size > OE_INT_MAX)
+        OE_RAISE(OE_INVALID_PARAMETER);
+
+    /* Get the NID for this curve type */
+    if ((nid = _get_nid(ec_type)) == NID_undef)
+        OE_RAISE(OE_FAILURE);
+
+    /* Create the public EC key */
+    {
```

Comment:
The bracket can be removed

Reply:
Removing this bracket causes the compilation to fail due to unreachable goto label